### DISCUSSION TITLE
tooth number detection and diagnostics (best  practics right Way)

### DISCUSSION THREAD

**Discussion by DimIsaev** [Project-MONAI/MONAILabel#1326] [General]
What is the best way to go in this issue of detection and training in determining diagnoses

Any help is appreciated!
Article, links, examples

thanks

**Comment by tangy5** [comment]
I feel this is a general question. We can keep track.

From MONAI Label side, most tasks are segmentation, delineat/segment tissues/targets. 
Now, it's extending to broader tasks. 

Andreas had a vertebra localization+segmentation app showcase the proactice, maybe it's helpful
https://github.com/Project-MONAI/MONAILabel/tree/main/sample-apps/radiology#multistage-vertebra-segmentation

**Reply by DimIsaev** [reply]
> У Андреаса было приложение локализации + сегментации позвонков, демонстрирующее практику, может быть, оно полезно https://github.com/Project-MONAI/MONAILabel/tree/main/sample-apps/radiology#multistage-vertebra-segmentation

gone to read and test
